src/crypto/CHIPCryptoPALPSA.h

```diff
@@ -62,7 +62,8 @@ enum class KeyIdBase : psa_key_id_t
 {
     Minimum     = CHIP_CONFIG_CRYPTO_PSA_KEY_ID_BASE,
     Operational = Minimum, ///< Base of the PSA key ID range for Node Operational Certificate private keys
-    Maximum     = Operational + kMaxValidFabricIndex,
+    DACPrivKey  = Operational + kMaxValidFabricIndex + 1,
```

Comment:
Isn't that risky, what if someone decide to increase number of  fabrics in the future with DFU? Should we keep DAC in first well-known position?

Reply:
Ahh.. it's the UINT8_MAX not the current max fabric.. Then it probably all is okay.

Reply:
So will be ok if the DACPrivKey is set to the Minimum value within that enum?

Reply:
I think it's all okay. I though that  `kMaaxValidFabricIndex` indicates the current max supported fabrics e.g. 5. But in fact it means 254.. so it's fine.